src/test/resources/pages/instructorFeedbackDoneEditing.html

```diff
@@ -751,7 +751,7 @@ <h5 class="col-sm-2">
                      </span>
                   </td>
                   <td class="session-response-for-test recent">
-                     <img src="/images/ajax-loader.gif"/>
+                     0 / 4
                                        </td>
```

Comment:
not actually part of the issue, but this needs to be fixed to ensure dev green. can this be included?